Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -826,63 +816,62 @@ private OrderStatus ConvertOrderStatus(Messages.Order orderDetails)
 
 
         /// <summary>
-        /// Gets all open positions
+        /// Gets all open postions and account holdings
         /// </summary>
         /// <returns></returns>
         public override List<Holding> GetAccountHoldings()
         {
             var holdingsList = new List<Holding>();
-            if (_zerodhaProductType.ToUpperInvariant() == KiteProductType.MIS.ToString().ToUpperInvariant() || _zerodhaProductType.ToUpperInvariant() == KiteProductType.NRML.ToString().ToUpperInvariant())
+            // get MIS and NRML Positions
+            if (string.IsNullOrEmpty(_zerodhaProductType) || _zerodhaProductType.ToUpperInvariant() == KiteProductType.MIS.ToString().ToUpperInvariant() || _zerodhaProductType.ToUpperInvariant() == KiteProductType.NRML.ToString().ToUpperInvariant())
```

Comment:
nit - line is a bit long? Can perform `_zerodhaProductType.ToUpperInvariant()` once I think?